book/src/07_parallel_join.md

```diff
@@ -0,0 +1,45 @@
+# [DRAFT] Parallel Join
+
+As mentioned in chapter dedicated on how to [dispatch][c3] systems
+Specs automatically parallises execution of systems that have
```

Comment:
IIRC I used British spelling so far, so we should keep it consistent (either everything with AE spelling or BE, I don't care).